modules/sbo-creating-a-postgresql-database-instance.adoc

```diff
@@ -0,0 +1,121 @@
+      repos:
+      - name: repo1
+        volume:
+          volumeClaimSpec:
+            accessModes:
+            - "ReadWriteOnce"
+            resources:
+              requests:
+                storage: 1Gi
+      - name: repo2
+        volume:
+          volumeClaimSpec:
+            accessModes:
+            - "ReadWriteOnce"
+            resources:
+              requests:
+                storage: 1Gi
+  proxy:
+    pgBouncer:
+        image: registry.developers.crunchydata.com/crunchydata/crunchy-pgbouncer:centos8-1.15-2
+EOD
+----
++
+The annotations added in this `PostgresCluster` CR help in enabling the service binding connection and trigger the Operator reconciliation.
++
+The output verifies that the database instance is created:
++
+.Example output
+----
+namespace/my-postgresql configured
+postgrescluster.postgres-operator.crunchydata.com/hippo created
+----
+
+. After the database instance is created, ensure that all the pods in `my-postgresql` namespace are running (it will take a few minutes):
++
+----
+$ oc get pods -n my-postgresql
+----
++
+The output verifies that the database is created:
++
+.Example output
+----
+NAME                                     READY   STATUS      RESTARTS   AGE
+hippo-backup-6th6--1-28849               0/1     Completed   0          1m
+hippo-instance1-sl4r-0                   2/2     Running     0          2m
+hippo-pgbouncer-8454474bc7-lhcn9         2/2     Running     0          2m
+----
++
+The database created is empty at this stage. You can set its schema and project a sample data set to interact with the sample application.
+
+. Initialize the database with the schema and sample data using the custom shell script by copying the following code into shell and executing it:
++
+----
+$ cat << EOD | bash
+#!/bin/bash
+
+export pgo_cluster_name=hippo
+export cluster_namespace=my-postgresql
+export pgo_cluster_username=hippo
+export PGPASSWORD=\$(oc -n "\${cluster_namespace}" get secrets \
+  "\${pgo_cluster_name}-pguser-\${pgo_cluster_username}" -o "jsonpath={.data['password']}" | base64 -d)
+nohup oc -n \${cluster_namespace} port-forward svc/hippo-pgbouncer 5432:5432 &
+sleep 5
+curl -LO https://raw.githubusercontent.com/spring-petclinic/spring-petclinic-rest/master/src/main/resources/db/postgresql/initDB.sql
+psql -h localhost -U "\${pgo_cluster_username}" "\${pgo_cluster_name}" -f initDB.sql
+curl -LO https://raw.githubusercontent.com/spring-petclinic/spring-petclinic-rest/master/src/main/resources/db/postgresql/populateDB.sql
+psql -h localhost -U "\${pgo_cluster_username}" "\${pgo_cluster_name}" -f populateDB.sql
+EOD
+----
++
+The output in the terminal shows you that the database is getting configured for the application.
+
+After the database is configured, you can initialize the Role-based access control (RBAC) support for the custom resources (CRs) of Crunchy PostgreSQL Operator by creating appropriate `ClusterRole` and `ClusterRoleBinding` resources.
```

Comment:
This section is obsolete and should be removed since crunchy postgres is supported by default via #1043. @pedjak please confirm